src/osd/PrimaryLogPG.h

```diff
@@ -1352,7 +1347,7 @@ class PrimaryLogPG : public PG, public PGBackend::Listener {
   // -- proxywrite --
   map<ceph_tid_t, ProxyWriteOpRef> proxywrite_ops;
 
-  void do_proxy_write(OpRequestRef op, const hobject_t& missing_oid);
+  void do_proxy_write(OpRequestRef op, const hobject_t& missing_oid, ObjectContextRef obc = NULL);
```

Comment:
this should get squashed iwth teh other parts of the first patch that you are making use of